composer/models/__init__.py

```diff
@@ -38,3 +38,4 @@
 from composer.models.unet import UnetHparams as UnetHparams
 from composer.models.vit_small_patch16 import ViTSmallPatch16 as ViTSmallPatch16
 from composer.models.vit_small_patch16 import ViTSmallPatch16Hparams as ViTSmallPatch16Hparams
+from composer.models.huggingface import HuggingFaceModel as HuggingFaceModel
```

Comment:
as much as it pains me as a software engineer, I'd vote for importing HuggingFaceModel in the first cell where it's used. I was going thru this and had to scroll up to look for where `HuggingFaceModel` came from when I first saw it.

Reply:
In the notebook? We can certainly do that for learning purposes

Reply:
yes